src/components/Portfolio/Portfolio.tsx

```diff
@@ -12,15 +13,16 @@ import { ChainCard, Frame, PortfolioHeader, PortfolioRoot } from './styleds';
 const Portfolio: React.FC = () => {
   const { account } = usePangolinWeb3();
   const { data: balances, isRefetching, isLoading } = useGetChainsBalances();
-  const [showBalances, setShowBalances] = useState(true);
+  const [showUserBalances, setUserShowBalances] = useShowBalancesManager();
+  const [showBalances, setShowBalances] = useState(showUserBalances);
```

Comment:
why do we need extra `useState` here when boolean is already coming from redux ?